### PR TITLE
added home page

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
Stellar_Landing_Page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1 +1,113 @@
...
+  text-align: center;
+  padding-top: 2rem;
+}
+
+.poster {
+  max-width: 100%;
+  margin: 0 auto;
+  align-content: center;
+  justify-content: center;
+  display: flex;
```
Comment (kjais1720) [inline index.css]: You can use utility class for this

File: index.css [hunk 2]
```diff
@@ -1 +1,113 @@
...
+  border: 1rem transparent;
+}
+
+.input-bar {
+  border: 1rem transparent;
+  width: 100%;
+}
+.home-container {
+  justify-content: center;
+  align-content: center;
```
Comment (kjais1720) [inline index.css]: same as above

File: index.css [hunk 3]
```diff
@@ -1 +1,113 @@
...
+
+.glass-icon-color {
+  font-size: 1.5rem;
+  color: var(--primary-color);
+}
+
+.search-container {
+  padding: 0rem;
+  align-content: center;
+  justify-content: center;
```
Comment (kjais1720) [inline index.css]: same as above

**Review by kjais1720 — CHANGES_REQUESTED** [review]

File: index.css [hunk 4]
```diff
@@ -1 +1,111 @@
...
+.search-container{
+    padding: 0rem;
+    align-content: center;
+    justify-content: center;
+    border-radius: 2rem;
+}
+
+.search-button{
+    border-radius: 0.2rem 0 0.2rem 0.2rem;
+    border:1rem transparent;
```
Comment (raman1999) [inline index.css]: try to use px for sizes less than 10px

File: index.css [hunk 5]
```diff
@@ -1 +1,111 @@
...
+    padding-top: 2rem;
+}
+
+.poster{
+max-width: 100%;
+margin:0 auto;
+align-content: center;
+justify-content: center;
+display: flex;
+}
```
Comment (raman1999) [inline index.css:31]: you can use util class  for this, as gonna be used mostly

File: index.css [hunk 6]
```diff
@@ -1 +1,111 @@
...
+.poster-container{
+   position: relative;
+}
+
+.banner-button{
+    position: absolute;
+    top:-13rem;
+    left:40rem;
+    width:8rem;
+    text-align: center;
```
Comment (raman1999) [inline index.css]: util class

**Review by raman1999 — COMMENTED** [review]
UI looks good . suggested minor changes

File: index.css (hunk 1 above)
Comment (Priyanshi-Tiwari) [reply index.css]: done

File: index.css (hunk 3 above)
Comment (Priyanshi-Tiwari) [reply index.css]: done

File: index.css (hunk 5 above)
Comment (Priyanshi-Tiwari) [reply index.css:31]: done

**Pull request merged**